docs/examples/hello_world/README.md

```diff
@@ -69,9 +69,9 @@ parameters:
 
 <2> Provisioner
 
-<3> GlusterFS defined EndPoint taken from Step 1 above (kubectl get endpoints)
+<3> GlusterFS defined EndPoint taken from Step 1 above (kubectl get endpoints). As of Kubernetes 1.7, this parameter should be removed as Kubernetes will reject this YAML definition.
```

Comment:
I updated it to `>= 1.6` -- thinking maybe it's appropriate as there's instructions relative to kube 1.5 above? Assuming that parameter would be necessary with Kube 1.5, worth keeping if there's 1.5 instructions? *shrug*